src/Program.cs

```diff
@@ -156,9 +104,20 @@ private static int Run(RunOptions opts)
                 project.Save();
             }
 
+            // Executes the command to download the version if it doesn't exist
+            if (Pull(new PullOptions()
```

Comment:
Can we extract the necessary functionality here into a method in `VersionManager`? We don't really need any of the docs logic etc. that calling this command provides so it might be good to pull this out to it's own method:
```
VersionManager.EnsureVersionInstalled(project.Runtime.Version, project.Runtime.Tier, project.Runtime.Platform)
```